src/components/Trade/Components/TradeQuotes/TradeQuote.tsx

```diff
@@ -113,11 +113,17 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
   if (!feeAsset)
     throw new Error(`TradeQuoteLoaded: no fee asset found for chainId ${sellAsset?.chainId}!`)
 
-  const networkFeeFiat = feeAssetFiatRate
-    ? bnOrZero(
-        fromBaseUnit(quote.steps[0].feeData.networkFeeCryptoBaseUnit, feeAsset.precision),
-      ).times(feeAssetFiatRate)
-    : undefined
+  const networkFeeCryptoBaseUnit = quote.steps[0].feeData.networkFeeCryptoBaseUnit
+
+  const networkFeeFiat = useMemo(
+    () =>
+      feeAssetFiatRate && networkFeeCryptoBaseUnit
+        ? bnOrZero(fromBaseUnit(networkFeeCryptoBaseUnit, feeAsset.precision))
+            .times(feeAssetFiatRate)
```

Comment:
Not related to this PR, but we may want to start using the `user` or `selectedCurrency` vernacular instead of `fiat` to make it clear this isn't necessarily a USD fiat rate, but the fiat rate in user-selected currency, see https://github.com/shapeshift/web/issues/4866

Reply:
absolutely, this is something i found confusing joining the org - fiat !== usd